.azure-pipelines/azure-pipelines.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  DATABASE_SCHEMA: 1.30.0
+  DATABASE_SCHEMA: 1.31.0
```

Comment:
Once merged this should (I think) trigger a PR to regenerate the sqlalchemy ORM for this database version.

Reply:
That is also my impression. Fingers crossed.